source/common/upstream/load_stats_reporter.cc

```diff
@@ -21,9 +27,12 @@ LoadStatsReporter::LoadStatsReporter(const LocalInfo::LocalInfo& local_info,
           Grpc::VersionedMethods("envoy.service.load_stats.v3.LoadReportingService.StreamLoadStats",
                                  "envoy.service.load_stats.v2.LoadReportingService.StreamLoadStats")
               .getMethodDescriptorForVersion(transport_api_version)),
-      time_source_(dispatcher.timeSource()) {
+      time_source_(dispatcher.timeSource()),
+      load_report_stats_store_(load_report_stats_store), send_request_latencies_{false} {
+  ASSERT(load_report_stats_store != nullptr);
   request_.mutable_node()->MergeFrom(local_info.node());
   request_.mutable_node()->add_client_features("envoy.lrs.supports_send_all_clusters");
+  request_.mutable_node()->add_client_features("envoy.lrs.supports_request_latency_percentiles");
```

Comment:
Do we actually need a client feature to indicate that the client supports this?  Or is it sufficient for the server to say it wants the data and understand that clients that don't support this won't send anything?

I'm not opposed to having a client feature here if it's useful; I'm just wondering what the use-case is and whether it's really necessary.

If we keep this, please document it in docs/root/api/client_features.rst.

Reply:
Yeah, I don't have a great reason to do this, was emulating the `send_all_clusters`  feature.

Reply:
In the case of `send_all_clusters`, the server would request that instead of explicitly listing the individual clusters it wants data for, so it needs to know whether the client supports that before it can use it, which is why we need the client feature.  But I don't think that's true for the latency feature; that's an additional thing that will either be supported or it won't, and the server probably doesn't need to know that in advance.

Reply:
yeah. got rid of it.